kong/kong.go

```diff
@@ -43,6 +43,7 @@ type Client struct {
 	Targets        *TargetService
 	Workspaces     *WorkspaceService
 	Admins         *AdminService
+	Users          *UserService
```

Comment:
Please rename Users to RBACUsers. Same applies to UserService.
User is generic enough to cause a confusion here.